tmk_core/common/chibios/syscall-fallbacks.c

```diff
@@ -18,6 +18,12 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 
+/* To compile the ChibiOS syscall stubs with picolibc
```

Comment:
Does this negate the need for the one in the `chconf.h`?

Reply:
Unfortunately not as this is the only way to inject it before the `syscall.c` of chibios is compiled